### PR TITLE
Feature/improve docs 2.0

### PR DESCRIPTION
**Pull request by PabloCorcho**

### CODE REVIEW

**Review by paranoya — APPROVED** [review]

File: docs/source/index.rst [hunk 1]
```diff
@@ -1,23 +1,35 @@
...
 
+PST supports a wide range of SSP models (e.g., **PopStar, XSL, MILES, BC03**) and offers flexible CEMs, allowing users to simulate observable quantities with precision. Additionally, it provides auxiliary tools to include **dust extinction** and **kinematics** effects.
 
+Key Features
+============
+- Intuitive interface to multiple SSP models (e.g., **PopStar**, **XSL**, **MILES**, **BC03**).
+- Implementation of various **Chemical Evolution Models (CEMs)** to simulate **Composite Stellar Populations (CSPs)**.
+- Tools to compute **synthetic spectra, photometry**, and **equivalent widths** from both SSPs and CSPs.
+- Optional integration of **dust extinction** and **kinematics** effects into the model outputs.
+- Modules for additional functionality, such as fitting **Spectral Energy Distributions (SEDs)** to observed data.
```
Comment (paranoya) [inline docs/source/index.rst:18]: Multiple SSPs and fitting are "future work"

File: docs/source/installation.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+.. code-block:: bash
+
+    pip install pst
+
+Alternatively, you can install it via `conda` if using Anaconda:
+
+.. code-block:: bash
+
+    conda install -c conda-forge pst
```
Comment (paranoya) [inline docs/source/installation.rst:25]: Mmm...

File: src/pst/utils.py [hunk 3]
```diff
@@ -50,24 +67,28 @@ def gaussian1d_conv(f, sigma, deltax):
...
+    """
+    isq = isinstance(quantity, u.Quantity)
+    if isq and default_unit is not None:
+        if not quantity.unit.is_equivalent(default_unit):
+            raise u.UnitTypeError(
+                "Input quantity does not have the appropriate units")
+        else:
+            return quantity
+    elif not isq and default_unit is not None:
+        return quantity * default_unit
```
Comment (paranoya) [inline src/pst/utils.py:89]: `quantity << default_unit`

File: src/pst/utils.py [hunk 4]
```diff
@@ -50,24 +67,28 @@ def gaussian1d_conv(f, sigma, deltax):
...
+    elif not isq and default_unit is None:
```
Comment (paranoya) [inline src/pst/utils.py:90]: What's the use case for `default_unit = None`?

**Review by paranoya — COMMENTED** [review]
Forget my concerns about the future and let's move forward

File: src/pst/utils.py (hunk 3 above)
Comment (PabloCorcho) [reply src/pst/utils.py:89]: Yes, I missed that

File: src/pst/utils.py (hunk 4 above)
Comment (PabloCorcho) [reply src/pst/utils.py:90]: That's actually true. It doesn't seem wise to check something that does not need to be checked...

**Pull request merged**